### ISSUE TITLE
docs: docker run command should map ports

### ISSUE THREAD

**Issue by skyzh** [singularity-data/risingwave#2647]
**Describe the bug**
```
docker run -it ghcr.io/singularity-data/risingwave:latest playground
```

is not enough. We should expose psql ports.

**To Reproduce**

**Expected behavior**

**Additional context**

**Comment by wangdexinhp** [comment]
Yes, the same problem occurred in my use. When I configured the Docker environment and started the Docker container, I could not use PSQL connection.
